### DISCUSSION TITLE
Bitbucket Server Support

### DISCUSSION THREAD

**Discussion by FaizanSh** [apache/devlake#4390] [Q&A]
Any updates on bitbucket Server integration? Lets connect and plan it out accordingly.

**Comment by klesh** [comment]
Work in progress, will be available soon.

**Reply by kiranzaman** [reply]
@klesh I am new to Apache DevLake, but would like to spare some time to help expedite the process/development. Please let me know if I could be of any help. 
Thank you!

**Reply by klesh** [reply]
Appreciated it, we are planning to code freeze next Wed(2013-2-22), may another week or two for the `alpha` internal testing, and then `beta` public testing hopefully at the end of this month.
It would be great if you could help us test the `beta`, please join our slack and reach out to @hezyin (Hezheng Yin)  if you are interested.

**Reply by hezyin** [reply]
@kiranzaman We appreciate the help! If you're interested in helping with `beta` testing, here's the link to join our Slack: https://join.slack.com/t/devlake-io/shared_invite/zt-17b6vuvps-x98pqseoUagM7EAmKC82xQ

**Comment by Startrekzky** [comment]
BitBucket server is still under development. The contributor is supporting the latest BitBucket Server v8.9.
Tracking ticket: https://github.com/apache/incubator-devlake/issues/4568
